### PR TITLE
Bump monorepo

### PR DESCRIPTION
**Pull request by alexfauquette**
Seems to be ok. One remaining issue: the slots

The monorepo script is using name convention `Slots` and `SlotProps` to detect is an interface is the typing of the slots.

For next branch, I solved this issue by migrating all our components to the same naming convention as core (replacing `SlotsComponentsProps` by `SlotProps` (1 stone 2 birds) but here I can't since it's a breaking change.

TO have this PR pass we should modify the monorepo such that it also accept our naming.

I can remember that their are some inconsistencies in our naming about the usage of `s` at the end of `SlotsComponents.`

### CODE REVIEW

File: docs/translations/api-docs/date-pickers/desktop-date-range-picker/desktop-date-range-picker.json [hunk 1]
```diff
@@ -163,6 +163,10 @@
     "RightArrowIcon": "Icon displayed in the right view switch button.",
     "SwitchViewButton": "Button displayed to switch between different calendar views.",
     "SwitchViewIcon": "Icon displayed in the SwitchViewButton. Rotated by 180° when the open view is 'year'.",
+    "DesktopPaper": "Custom component for the paper rendered inside the desktop picker's Popper.",
+    "DesktopTransition": "Custom component for the desktop popper [Transition](https://mui.com/material-ui/transitions/).",
+    "DesktopTrapFocus": "Custom component for trapping the focus inside the views on desktop.",
+    "Popper": "Custom component for the popper inside which the views are rendered on desktop.",
     "ActionBar": "Custom component for the action bar, it is placed below the picker views.",
```
Comment (alexfauquette) [inline docs/translations/api-docs/date-pickers/desktop-date-range-picker/desktop-date-range-picker.json]: Key order got modified, I don't know why

**Review by LukasTy — APPROVED** [review]
LGTM.
Great work! 👍 💯

File: packages/grid/x-data-grid-pro/src/tests/components.DataGridPro.test.tsx [hunk 2]
```diff
@@ -33,11 +33,11 @@ describe('<DataGridPro/> - Components', () => {
       expect(document.querySelector(`.${gridClasses.rowCount}`)).to.equal(null);
     });
 
-    it('should throw a console error if hideFooterRowCount is used with pagination', () => {
-      expect(() => render(<TestCase hideFooterRowCount pagination />)).toErrorDev(
-        'MUI: The `hideFooterRowCount` prop has no effect when the pagination is enabled.',
-      );
-    });
+    // it('should throw a console error if hideFooterRowCount is used with pagination', () => {
```
[the remark this answers is not in the document]
Comment (alexfauquette) [reply packages/grid/x-data-grid-pro/src/tests/components.DataGridPro.test.tsx:36]: It's a cherry pick from next. There is an intent to do it (on next only). data grid team said they will have a look to put that outside of proptyes

File: scripts/buildApiDocs/tsconfig.json
[the remark this answers is not in the document]
Comment (alexfauquette) [reply scripts/buildApiDocs/tsconfig.json:1]: Good question, I will try

File: packages/grid/x-data-grid-pro/src/tests/components.DataGridPro.test.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (LukasTy) [reply packages/grid/x-data-grid-pro/src/tests/components.DataGridPro.test.tsx:36]: Gotcha, thanks for clarifying. 👌

**Pull request merged**